lib/omniauth-dsds/spec/sign_in_helper.rb

```diff
@@ -15,9 +15,9 @@ def mock_token(token: "ABCDEF")
           })
         end
 
-        def mock_profile(token: "ABCDEF", profile: {}, authentication_site_url: "http://app.example.com")
-
-          response_body = profile.reverse_merge({
+        def mock_profile(token: "ABCDEF", options: {}, authentication_site_url: "http://app.example.com")
```

Comment:
token param is not used anywhere.... shouldn't it be passed inside an Authorization header?